packages/airnode-node/src/evm/verification/request-verification.ts

```diff
@@ -45,6 +51,8 @@ export function verifyRrpTriggers(
 ): LogsData<Request<ApiCall>[]> {
   const logsWithVerifiedApiCalls: LogsData<Request<ApiCall>>[] = apiCalls.map((apiCall) => {
     if (apiCall.status !== RequestStatus.Pending) {
+      // TODO: Do we need to log this? We do not follow the same practice in applySponsorAndSponsorWalletRequestLimit
```

Comment:
It's a debug log so I would say it's better to log too much information than too little, most users will never see it. I would update the other function to be consistent

Reply:
I think this needs to be checked in other places for consistency as well. I am also not sure whether we should not ignore those requests directly.

I've added a bunch of TODOs https://github.com/api3dao/airnode/pull/724/files#diff-bafef992d17b51ddeeecf5c4f67a79342f80c0b28bd383c18310ce0da1a57278R43 and created to https://api3dao.atlassian.net/browse/AN-433 resolve this.

I'd revisit this properly once there is more time.